docs/website/references/cli/defradb_client_p2p_collection_list.md

```
@@ -1,20 +1,20 @@
+## defradb client p2p collection list
 
-Get all P2P collections
+List P2P collections
 
 ### Synopsis
 
-Get all P2P collections in the pubsub topics.
+List P2P collections in the pubsub topics.
 This is the list of collections of the node that are synchronized on the pubsub network.
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)